modules/rostests/winetests/msvcrt/environ.c

```diff
@@ -233,12 +233,12 @@ static void test__wenviron(void)
         wchar_t ***retptr = p__p___winitenv();
         wchar_t *value;
 
-        ok( *retptr != NULL, "Expected *__p___winitenv() to be NULL\n" );
+        skip_2k3_fail ok( *retptr != NULL, "Expected *__p___winitenv() to be NULL\n" );
```

Comment:
hmm... Isn't it "expected .... to NOT be NULL" ?

Reply:
The test succeeds on Windows 10, it fails on Windows 2003. So it's either this or that.